README.md

```diff
@@ -7,32 +7,27 @@ Runs an [Exhibitor](https://github.com/Netflix/exhibitor)-managed [ZooKeeper](ht
 docker build -t <tag> .
 ```
 
-###### Create the Security group
-```
-GROUP_ID=$(aws ec2 create-security-group --group-name app-exhibitor --description "Exhibitor security group" \
-  | sed -rn 's/.*GroupId": "(.*)".*/\1/p' | tail -n 1)
-aws ec2 authorize-security-group-ingress --group-id $GROUP_ID --protocol tcp --port 22 --cidr 0.0.0.0/0
-aws ec2 authorize-security-group-ingress --group-id $GROUP_ID --protocol tcp --port 80 --cidr 0.0.0.0/0
-aws ec2 authorize-security-group-ingress --group-id $GROUP_ID --protocol tcp --port 2181 --cidr 0.0.0.0/0
-aws ec2 authorize-security-group-ingress --group-id $GROUP_ID --protocol tcp --port 2888 --cidr 0.0.0.0/0
-aws ec2 authorize-security-group-ingress --group-id $GROUP_ID --protocol tcp --port 3888 --cidr 0.0.0.0/0
-aws ec2 authorize-security-group-ingress --group-id $GROUP_ID --protocol tcp --port 8181 --cidr 0.0.0.0/0
-```
-
 ###### Create S3 bucket
 ```
-S3_BUCKET="exhibitor-app"
+S3_BUCKET="exhibitor-bucket"
 aws s3 mb s3://$S3_BUCKET
 ```
 
 ###### Register appliance in YourTourn
 Docs: http://docs.stups.io/en/latest/components/yourturn.html
 
+Make sure that your got an unique ```APPLICATION_ID```
+
 ###### Deploy with Senza
 ```
-senza create exhibitor-appliance.yaml <STACK_VERSION> <DOCKER_IMAGE> $GROUP_ID <HOSTED_ZONE> $S3_BUCKET <MINT_BUCKET> <SCALYR_KEY>
+senza create exhibitor-appliance.yaml <STACK_VERSION> <APPLICATION_ID> <DOCKER_IMAGE_WITH_VERSION_TAG> <HOSTED_ZONE> $S3_BUCKET <MINT_BUCKET> <SCALYR_KEY> [--region AWS_REGION]
+```
+
+A real world example would be:
+```
+senza create exhibitor-appliance.yaml 1 saiki-exhibitor pierone.example.org/myteam/exhibitor:0.1-SNAPSHOT example.org. exhibitor-bucket example-stups-mint-some_id-eu-west-1 some_scalyr_key --region eu-west-1
 ```
 
-Cloudformation stack will start 3 EC2 instances in autoscaling group and create internal load balancer in front of EC2 instances. Also it will create DNS record ```"<STACK_VERSION>.exhibitor.<HOSTED_ZONE>"``` which points to a load balancer.
+Cloudformation stack will start 3 EC2 instances in autoscaling group and create internal load balancer in front of EC2 instances. Also it will create DNS record ```"<STACK_NAME>-<STACK_VERSION>.<HOSTED_ZONE>"``` which points to a load balancer.
```

Comment:
aha, i see, we take different usage of stack name and its version... such as, we named stack name like "exhibitor-postgres" or "exhibitor-spilo", and version we used something like: "prod1", "test2", "1", "2" etc...

actually question is only, if version string must be unique for all exhibitor-appliances, or the combination stack name+stack version is unique should be already enough.

such as, for us, exhibitor-postgres has version test1, exhibitor-spilo can also has version test1.

and it seems you used exhibitor as stack name and it has version postgrestest1 and spilotest1 (bad is no delimiter allowed in version name...)